### ISSUE TITLE
Problem with input in Shugaku puzzles

### ISSUE THREAD

**Issue by jrunning** [robx/pzprjs#261]
Very often in a Shugaku puzzle when placing a futon it will appear shaded red, and when placing another futon the red futon either partially or fully disappears. This problem has been happening on the puzz.link site for a few months. Here is a GIF of the problem:

![Screen Recording 2022-05-26 at 1 31 15 PM](https://user-images.githubusercontent.com/117983/170553595-fa1b366e-e8da-42a8-9d15-8fbc54bdd52b.gif)

This issue may be related to #61, however it happens very frequently in every Shugaku puzzle, however unpredictably.

**Comment by robx** [comment]
Yes, I think it's the same as #61, but we can keep this issue open instead. I also see it now and then, and have no idea what's happening.
